src/pal/src/cruntime/printfcpp.cpp

```diff
@@ -1471,7 +1456,7 @@ int CoreVfwprintf(CPalThread *pthrCurrent, PAL_FILE *stream, const wchar_16 *for
                     va_list apcopy;
 
                     va_copy(apcopy, ap);
-                    TempInt = NativeVsnprintf(pthrCurrent, TempSprintfStr, TEMP_COUNT, TempBuff, apcopy);
+                    TempInt = vsnprintf(TempSprintfStr, TEMP_COUNT, TempBuff, apcopy);
```

Comment:
Should we use external hook fallback for standard C99 `snprintf` (supported by VS2015, Clang, GCC) like this: https://github.com/jasonwilliams200OK/libuv/commit/43be2a1ae65adac9cbd56ba31061701679b3555d? Or drop support for `_MSC_VER<1900` altogether and stick to only `snprintf`?
@janvorli, @jkotas